src/condor_negotiator.V6/matchmaker.cpp

```diff
@@ -2747,43 +2758,16 @@ trimStartdAds_PreemptionLogic(ClassAdListDoesNotDeleteAds &startdAds) const
 		return removed;
 	}
 
-	bool is_pslot = false;
-	startdAds.Open();
-	while( (ad=startdAds.Next()) ) {
-		if(ad->LookupString(ATTR_STATE, curState, sizeof(curState))) {
-			if ( strcmp(curState,claimed_state_str)==0
-			     || strcmp(curState,preempting_state_str)==0)
-			{
-				// Ignore Claimed pslots, they aren't eligible for preemption
-				ad->LookupBool(ATTR_SLOT_PARTITIONABLE, is_pslot);
-				if (is_pslot) {
-					continue;
-				}
-				startdAds.Remove(ad);
-				removed++;
-			}
-		}
-	}
-	startdAds.Close();
-
-	dprintf(D_FULLDEBUG,
-			"Trimmed out %d startd ads due to NEGOTIATOR_CONSIDER_PREEMPTION=False\n",
-			removed);
-
```

Comment:
Why did you remove this block? It's how we ignore claimed slots when preemption is disabled.